frontend/src/views/web/contact-us.html

```diff
@@ -0,0 +1,60 @@
+<div class="white-bg view-container">
+    <main-header></main-header>
+    
+    <div class="grad-container">
+        <div class="row">
+            <div class="col s12 m8 l8">
+                    <br><br>
+                    <h4 align="center" class="text-dark-black"><strong>Contact Us</strong></h4>
+                    <form name="contactusForm" ng-submit="" id="contact-us" novalidate>
+        			<!-- Name -->
+        				<div class="input-field align-left">
+            				<input type="text" id="name" name="name" ng-minlength="3" required>
+            				<span class="form-icon"><i class="fa fa-user"></i></span>
+            				<label for="name">Name *</label>
+            				<div class="wrn-msg text-highlight" ng-messages="contactusForm.name.$error" ng-if="contactusForm.name.$touched || contactusForm.$submitted">
+                				<p ng-message="minlength">name is too short.</p>
+                				<p ng-message="required">name is required.</p>
+            				</div>
+        				</div>
+        			<!-- Email -->
+        				<div class="input-field align-left">
+            				<input type="text" id="email" name="email" required>
+            				<span class="form-icon"><i class="fa fa-envelope"></i></span>
+            				<label for="email">Email *</label>
+        				</div>
+        			<!-- Message -->
+        				<div class="input-field align-left">
+            				<input type="text" id="message" name="message" required>
```

Comment:
@Ram81 : use `<textarea></textarea>` for 'message' instead of `<input>` since details shouldn't be inline.